src/main/java/com/github/alexthe666/iceandfire/event/ServerEvents.java

```diff
@@ -555,18 +595,12 @@ public void onLivingHurt(LivingHurtEvent event) {
     				}
     				break;
     			case 2:
-    				if(target.getCreatureAttribute() != EnumCreatureAttribute.ARTHROPOD || target instanceof EntityDeathWorm) {
```

Comment:
Can you revert this change (and presumably the corresponding change in ItemRegistry) so it'll be easier to merge with my most recent change on the main branch? (Don't merge that commit in to this manually, though.)